src/Runner/Runner.php

```diff
@@ -134,8 +134,16 @@ public function fix()
             Tokens::clearCache();
 
             if ($fixInfo) {
-                $name = $this->directory->getRelativePathTo($file);
-                $changed[$name] = $fixInfo;
+                $path = $file->getPathname();
+
+                /* workaround for Symfony/Finder setting doubled slash for root folders in
+                SplFileInfo. See `RecursiveDirectoryIterator::current()`. */
```

Comment:
is it known bug of `Symfony/Finder` ? if so, please link a issue.
if not, please make an issue and link it.